libraries/standard/mqtt/cbmc/proofs/MQTT_DeserializeAck/Makefile

```diff
@@ -30,8 +30,7 @@ DEFINES += -DREMAINING_LENGTH_MAX=$(REMAINING_LENGTH_MAX)
 INCLUDES +=
 
 REMOVE_FUNCTION_BODY +=
-# The loop is unwound REMAINING_LENGTH_MAX + 1 times.
-UNWINDSET += __CPROVER_file_local_mqtt_lightweight_c_readSubackStatus.0:4
+UNWINDSET += __CPROVER_file_local_mqtt_lightweight_c_readSubackStatus.0:$(REMAINING_LENGTH_MAX)
```

Comment:
This unwinds the loop $(REMAINING_LENGTH_MAX) times, not +1

Reply:
Yes I deleted that comment, thanks.

Reply:
For the remainingLength it is used in a < expression and therefore follows the k+1 pattern you mentioned above.